DebarshiB13/mytunes
# Feat/itunes-tunecard UI

### Ticket Link

---

### Related Links

---

### Description

---
Second PR for the Itunes app.

The new changes are :
Implementation of the UI part.
List of songs displayed from search results in form of Grid
Each song has a card.
Play/Stop songs
Unit tests for each item.

### Steps to Reproduce / Test

---
 * Go to localhost:3000/songs.
 * Type any song or artist name on the search bar.
 * List of cards/ error will appear based on the response.
 * Play/stop any song by clicking the play/stop button.

### GIF's

---
![feat_tunecard_ui](https://user-images.githubusercontent.com/89914287/132449418-0726fdd5-7698-4e3c-9d2d-a4e065ad86b7.gif)
![feat_tunecard_ui_test](https://user-images.githubusercontent.com/89914287/132449439-d457ddad-c814-4334-8bc6-1b84080dbc19.gif)
